### ISSUE TITLE
Start pulling from LimsRest (after modeling is *complete*)

### ISSUE THREAD

**Issue by ao508** [mskcc/cmo-metadb#71]
Start with requests from Jan 2017 +

Before actually beginning the full persistence process of LimsRest request to graphdb:
- Let's start with first 50 requests from 2017 and estimate how long it might take? If we could infer how long more requests take to fetch from LimsRest and persist to graphdb then that may help us in scheduling our jobs for populating graphdb. 

Next:
- Part of this effort depends on #141 (modifying the publisher to accept a timestamp in milliseconds and start pulling requests based off of that)
- Keep in mind that LimsRest server has scheduled downtime daily at ~2-3am. Downtime is only about 5-10 minutes. The script or tool we put together to start populating the graphdb should take this into consideration. 
- Can we get all the request ids and timestamps and then "organize" or sort them in some way to measure progress on requests from 2017, 2018, etc.? Can we put together a more elegant approach for this? Pass a year filter to the publisher tool to not process anything outside of whatever year we specify?

**PROGRESS**
- [x] 2017 backfill
- [x] 2018 backfill
- [x] 2019 backfill (in progress 6/7/21)
- [x] 2020 backfill
- [x] 2021 backfill

If server is down then should tool wait for 10 minutes before continuing? If connection fails 3 times after a 10-min wait period before each attempt to reconnect then save the remaining request ids into some file and exit gracefully.

**Comment by ao508** [comment]
We can start this after the patient aliases node PR has been merged into master.
